selection_iteration/__openerp__.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+#   Module for OpenERP
+#   Copyright (C) 2015 Akretion (http://www.akretion.com).
+#   @author Valentin CHEMIERE <valentin.chemiere@akretion.com>
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as
+#   published by the Free Software Foundation, either version 3 of the
+#   License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+{
+    'name': 'Selection iteration',
+    'version': '0.0.1',
+    'author': 'Akretion',
+    'website': 'www.akretion.com',
+    'license': 'AGPL-3',
+    'category': 'Generic Modules',
+    'description': """
+    Allow to iterate on selection values with a button.\n\n
+    - The model must inherit from 'abstract.selection.rotate'\n
+    - override _get_values_from_selection method to match with
+       your selection field \n
+    - Just create a button like:\n
+    <button name="iter_selection_next" string="Next Value"
+       context="{'selection_field': '$selection_field'}" type="object"
+       icon="gtk-go-forward"/>
+
+    For more information see the example in the module.
+    """,
```

Comment:
Can you extract the description to ause the template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst